app/src/main/java/com/example/tasky/common/presentation/AgendaCard.kt

```diff
@@ -103,10 +118,25 @@ fun AgendaCard(
 }
 
 @Composable
-fun CustomCheckbox(isChecked: Boolean, color: Color, size: Dp) {
-    IconButton(onClick = { /*TODO*/ }) {
+fun CustomCheckbox(
```

Comment:
I'd never name Composables starting with `Custom`, since that doesn't tell the caller anything about its purpose and in how far it's custom. In the end, every Composable is custom, isn't it? 😄  
If you're creating a Composable that represents a default styled component used across the project like here, I like to communicate that with the name instead. For example, `TaskyTextField`. That at least communicates that it's a component with the default style from the mockups which should be used for all components of this type across the project.